### PR TITLE
Fix getting started in readme

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -48,51 +48,92 @@ NB: you can also download MeiliSearch from **Homebrew** or **APT**.
...
+    // Create an index if your index does not already exist
+    _, err := client.Indexes().Create(meilisearch.CreateIndexRequest{
+        UID: "books",
+    })
+    if err != nil {
+        fmt.Println(err)
+        os.Exit(1)
+    }
+
+    documents := []Book{
```
Comment (qdequele) [inline readme.md]: Instead of creating your struct Book, you can use the JSON type: `map[string]interface{}`. It's just a proposition

Comment (curquiza) [reply readme.md]: If I try that:
```go
documents := map[string]interface{}{
		{BookID: 123, Title: "Pride and Prejudice"},
		{BookID: 456, Title: "Le Petit Prince"},
		{BookID: 1, Title: "Alice In Wonderland"},
		{BookID: 1344, Title: "The Hobbit"},
		{BookID: 4, Title: "Harry Potter and the Half-Blood Prince"},
		{BookID: 42, Title: "The Hitchhiker's Guide to the Galaxy"},
	}

updateRes, err := client.Documents("indexUID").AddOrUpdate(documents) // => { "updateId": 0 }
if err != nil {
        fmt.Println(err)
	os.Exit(1)
}

fmt.Println(updateRes.UpdateID)
```

I get:
```bash
./test.go:38:3: missing type in composite literal
```

Comment (qdequele) [reply readme.md]: I was thinking about that.

```go
documents := []map[string]interface{}{
		{BookID: 123, Title: "Pride and Prejudice"},
		{BookID: 456, Title: "Le Petit Prince"},
		{BookID: 1, Title: "Alice In Wonderland"},
		{BookID: 1344, Title: "The Hobbit"},
		{BookID: 4, Title: "Harry Potter and the Half-Blood Prince"},
		{BookID: 42, Title: "The Hitchhiker's Guide to the Galaxy"},
	}

updateRes, err := client.Documents("indexUID").AddOrUpdate(documents) // => { "updateId": 0 }
if err != nil {
        fmt.Println(err)
	os.Exit(1)
}

fmt.Println(updateRes.UpdateID)
```

Comment (curquiza) [reply readme.md]: Thanks! :)
(I had to chage `BookID` into `"book_id"` and `Title` into `"title"` as well)

File: readme.md [hunk 2]
```diff
@@ -150,16 +186,11 @@ err = client.Documents("books").List(ListDocumentsRequest{
...
 
 ```go
-documents := map[int]string{
-    { book_id: 123,  title: "Pride and Prejudice" },
-    { book_id: 456,  title: "Le Petit Prince" },
-    { book_id: 1,    title: "Alice In Wonderland" },
-    { book_id: 1344, title: "The Hobbit" },
-    { book_id: 4,    title: "Harry Potter and the Half-Blood Prince" },
-    { book_id: 42,   title: "The Hitchhiker\'s Guide to the Galaxy" }
+documents := []Book{
```
Comment (qdequele) [inline readme.md]: Warning the Book didn't exist anymore
```suggestion
documents := []map[string]interface{}{
```

Comment (curquiza) [reply readme.md]: oooooops 😅 thanks !

**Pull request merged**